components/ProductDisplay.js

```diff
@@ -0,0 +1,60 @@
+app.component('product-display',{
+    props:{
+        premium:{
+            type:Boolean,
+            required:true
+        }
+    },
+    template: 
+    /*html*/
+    `<div class="product-display">
+        <div class="product-container">
+          <product-image :image="image"></product-image>
```

Comment:
please make self closing the component

Reply:
also the prop would make more sense if it was named path or url

Reply:
done!